examples/nested/dir/BUILD

```diff
@@ -4,8 +4,8 @@ load("//sass:sass.bzl", "sass_binary")
 
 # Import our shared colors and fonts so we can generate a CSS file.
 sass_binary(
-    name = "nested",
-    src = "dir/main.scss",
+    name = "dir",
```

Comment:
I ran gazelle over these directories. If you would like them restored to their former glory because you were using them as tests/examples of functionality, just say so.